package-lock.json

```diff
@@ -0,0 +1,3271 @@
+{
```

Comment:
difficult to keep track of but this repo uses `bun` so technically no need for a `package-lock.json` (to be honest, I wish bun just used package lock but 🤷 )